README.md

```diff
@@ -20,166 +20,176 @@ import pyotp
 from logzero import logger
 
 api_key = 'Your Api Key'
-clientId = 'Your Client Id'
-pwd = 'Your Pin'
+username = 'Your client code'
+pwd = 'Your pin'
 smartApi = SmartConnect(api_key)
-token = "Your QR code value"
-totp=pyotp.TOTP(token).now()
-correlation_id = "abc123"
-
-# login api call
-
-data = smartApi.generateSession(clientId, pwd, totp)
-# logger.info(f"data: {data}")
-authToken = data['data']['jwtToken']
-refreshToken = data['data']['refreshToken']
-
-# fetch the feedtoken
-feedToken = smartApi.getfeedToken()
+try:
+    token = "Your QR value"
+    totp = pyotp.TOTP(token).now()
+except Exception as e:
+    logger.error("Invalid Token: The provided token is not valid.")
+    raise e
 
-# fetch User Profile
-res = smartApi.getProfile(refreshToken)
-smartApi.generateToken(refreshToken)
-res=res['data']['exchanges']
+correlation_id = "abcde"
+data = smartApi.generateSession(username, pwd, totp)
 
-#place order
-try:
-    orderparams = {
-        "variety": "NORMAL",
-        "tradingsymbol": "SBIN-EQ",
-        "symboltoken": "3045",
-        "transactiontype": "BUY",
+if data['status'] == False:
+    logger.error(f"Invalid Credentials {data}")
```

Comment:
This condition should check error codes as well.

Reply:
passing direct response body